### ISSUE TITLE
The keyboard focus moves inside the properties section even when it is collapsed: A11y_Azure Tools Storage Explorer_Blob Container_Properties tab_Focus Order

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8412]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags:

#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Keyboard; #WCAG2.4.3; #FTP; 

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch Azure Storage Explorer application.
2. TAB till "Account Management" in left Navigation and press ENTER key.
3. TAB till any "Subsection" for example 'azure tools Manual testing 1' and select it.
4. TAB till "Open explorer" button and press ENTER key
5. "Explorer" section will open. TAB till selected subscription and expand it
6. TAB till storage account and expand it.
7. TAB till any storage account and expand it.
8. TAB till "Blob Containers" and press ENTER key to expand it.
9. TAB till any "Blob container" and press ENTER key.
10 TAB till Properties tab present at left bottom of the screen and select it
11. Collapsed the Properties section and navigate with TAB key
12. Observe that the keyboard focus moves inside the properties section even when it is collapsed.

## Actual Experience
The keyboard focus moves inside the properties section even when it is collapsed

## Expected Experience
When properties section is in collapsed state keyboard should not move inside properties section.

## User impact
Keyboard only user will face difficulty if they press TAB key and keyboard focus is not viable.

## Attachment
![Image](https://github.com/user-attachments/assets/eb97ba42-a7ef-4eae-b52c-ab67c6f544cd)

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by craxal** [comment]
Reason for this is because the tab contents isn't actually hidden. It's just pushed off screen. We just need to set `display: none` for tab content when the footer panel is collapsed.

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kapilvaishna** [comment]
@craxal issue is repro in build 1.37.0 (104). PFA for reference  

![Image](https://github.com/user-attachments/assets/34472359-4613-41e5-b2cf-3d090cfd9934)

**Comment by craxal** [comment]
@kapilvaishna Sorry, I forgot to send you a link to the new build. The build number you should have should be at least 20250128.2 or later. In the future, can you please report the build number that you test on?

**Comment by kupatkar99** [comment]
Verified the issue in below Environment:
Azure Storage Explorer version:1.38.0-dev(104)
Issue is fixed. Hence Closing the bug. Please find below added attachment for reference.

![Image](https://github.com/user-attachments/assets/409b7df4-0d92-4c0d-91bf-840451dd326a)
